flink-1.17-connector-bigquery/flink-connector-bigquery/src/test/java/com/google/cloud/flink/bigquery/table/BigQueryDynamicTableFactoryTest.java

```diff
@@ -95,6 +103,42 @@ public void testBigQueryReadProperties() throws IOException {
         assertThat(actual.hashCode()).isEqualTo(expected.hashCode());
     }
 
+    @Test
+    public void testBigQueryUnboundedReadProperties() throws IOException {
+        Map<String, String> properties = getRequiredOptions();
+        properties.put(BigQueryConnectorOptions.COLUMNS_PROJECTION.key(), "aaa,bbb");
+        properties.put(BigQueryConnectorOptions.MAX_STREAM_COUNT.key(), "100");
+        properties.put(
+                BigQueryConnectorOptions.ROW_RESTRICTION.key(), "aaa > 10 AND NOT bbb IS NULL");
+        properties.put(
+                BigQueryConnectorOptions.SNAPSHOT_TIMESTAMP.key(),
+                "" + Instant.EPOCH.toEpochMilli());
```

Comment:
Use `Long.toString()` for converting long to string

Reply:
Fixed.